src/sentry/integrations/jira/client.py

```diff
@@ -156,10 +166,14 @@ def get_priorities(self):
         return self.get_cached(self.PRIORITIES_URL)
 
     def get_users_for_project(self, project):
-        return self.get(self.USERS_URL, params={'project': project})
+        # Jira Server wants a project key, while cloud is indifferent.
```

Comment:
wondering if it's worth trying to avoid the extra request for cloud given people already complain about how slow this can be. i guess `get_projects_list` is cached at least, so just a thought

Reply:
I can circle back on adding caching (APP-864).